compile.sh

```diff
@@ -64,23 +71,17 @@ if [[ -d $DEST/output ]]; then
 	read
 fi
 
-if [[ $EUID != 0 ]]; then
-	echo -e "[\e[0;35m warn \x1B[0m] This script requires root privileges"
-	sudo "$0" "$@"
-	exit 1
-fi
-
 #--------------------------------------------------------------------------------------------------------------------------------
 # Get updates of the main build libraries
 #--------------------------------------------------------------------------------------------------------------------------------
 [[ $(dpkg-query -W -f='${db:Status-Abbrev}\n' git 2>/dev/null) != *ii* ]] && \
 	apt-get -qq -y --no-install-recommends install git
 
-if [[ ! -d $SRC/lib ]]; then
+if [[ $EUID != 0 && ! -d $SRC/lib ]]; then
 	git clone https://github.com/igorpecovnik/lib
```

Comment:
So users who launch this script as root or via sudo won't be able to clone or update anything.